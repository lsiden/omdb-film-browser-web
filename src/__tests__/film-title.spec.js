import React from "react"
import { shallow } from "enzyme"

import FilmTitle from "film-title"

const films = require("./films.json").Search

const defaultProps = () => ({
  filmSummary: films[0],
})

const createWrapper = (props = {}) =>
  shallow(<FilmTitle {...{ ...defaultProps(), ...props }} />)

test("click on title invokes dispatchViewDetail()", () => {
  expect(createWrapper()).toMatchSnapshot()
})
