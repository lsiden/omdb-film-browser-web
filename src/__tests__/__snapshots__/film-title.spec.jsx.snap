// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`click on title invokes dispatchViewDetail() 1`] = `
<div>
  <a
    href="/film/tt2310332"
    onClick={[Function]}
    style={
      Object {
        "color": "#210d87",
      }
    }
    title="View film details"
  >
    The Hobbit: The Battle of the Five Armies, 2014
  </a>
</div>
`;
